packages/core/interfaces/src/future.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ArrayType,
+  ArrowType,
+  LabeledProductType,
+  NilType,
+  OptionType,
+  ScalarType,
+  SimpleTypes,
+  StructType,
+  TopType,
+  UnlabeledProductType,
+} from "@fluencelabs/interfaces";
+import { Call, Pipe, Objects, Tuples, Unions, Fn } from "hotscript";
```

Comment:
In CLI repo I have `verbatimModuleSyntax` in tsconfig and it asks to use `import type` when you import just type. You can read about this property here https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax and decide for yourself whethere you wanna have it or not. For CLI this was just needed because without it there was additional unneded imports and some stuff broke because of this lack of explicitness. With `type` modifiers you always make it explicit for typescript why and how the import is used. Here I don't have a strict preference, just for CLI repo it was useful at some point you can skip this rule

Reply:
I read about this property and really like it. Will try to update

Reply:
But i think that it should be done in a separate task because i need to review a lot of imports

Reply:
https://linear.app/fluence/issue/DXJ-494/add-verbatimmodulesyntax-property-to-tsconfig